frame/bounties/src/lib.rs

```diff
@@ -151,6 +158,12 @@ pub struct Bounty<AccountId, Balance, BlockNumber> {
 	bond: Balance,
 	/// The status of this bounty.
 	status: BountyStatus<AccountId, BlockNumber>,
+	/// Subbounties counter
+	subbounty_count: BountyIndex,
```

Comment:
You shouldn't need this. Just use `BountyCount` for the index of sub bounty. Makes everything simpler.

Reply:
Yes removed, reusing BountyCount for generating subbounty id.